WebDriverAgentTests/IntegrationTests/FBW3CTouchActionsIntegrationTests.m

```diff
@@ -315,26 +322,6 @@ - (void)testDoubleTap
   [self verifyGesture:gesture orientation:UIDeviceOrientationLandscapeLeft];
 }
 
-- (void)testLongPressWithCombinedPause
```

Comment:
I remember this did not work with Xcode 13 (even with iOS 14, that worked with Xcode 12.5)
so, I needed to skip this test for now for Xcode 13 env...
The reason was the action did not work well "after the screen rotation" in a simulator.

Let be bring this back and skip on Xcode 13

Reply:
I don't remember well just now, but probably the same steps worked on a real device via xcuitest-driver. (not this test itself). So, this rotation issue could be only on simulators.